CONTRIBUTING.md

```diff
@@ -229,7 +229,7 @@ We use the following label scheme to categorize issues:
 
 All issues should have a `type` label. `bug`/`feature`/`question`/`discussion` are self-explanatory. `dependencies` is for keeping package dependencies up to date. `maintenance` is a catch-all for any kind of cleanup or refactoring.
 
-They should also have one or more `area`/`status` labels. We use these labels to filter issues down so we can easily see all of the issues for a particular area, and keep the total number of open issues under control.
+They should also have one or more `area`/`status` labels. We use these labels to filter issues down so we can see all of the issues for a particular area, and keep the total number of open issues under control.
```

Comment:
Do you think this instance is unhelpful @carolstran? Although I am onboard with this PR generally (and personally am constantly fighting my inclination to use this unhelpful kind of language), I think in this case the "easily" is actually expressing something with some meaning.

Not to nitpick, it's not a big deal if we drop it, I'm just interested.

Reply:
My working theory is to always assume that things aren't easy - so even if you're using labels for this purpose, someone who is brand new to GitHub, open source, etc. may not be able to do the same. However, reading this instance back, this statement is talking about (I'd assume) the maintainers who a task like this would be easy... so I think you're right that this works here.

The PR is already merged, but if you'd like me to add it back - I could do that! Thanks for flagging because context and who you're addressing in each statement is important in these cases.

Reply:
Really, as I said I don't think in this particular instance it really matters, I was just more interested in the thought process. I guess it is true that there's no need to make a new maintainer feel like they are not one of "us" (i.e. the "we" in the sentence) because they don't feel it is easy to do what "we" do.

Probably "more easily" or "so we have a way to ..." would make more sense in this one case.